tests/test_ztf.rs

```diff
@@ -255,47 +265,112 @@ async fn test_process_ztf_lsst_xmatch() {
         .get_array("LSST")
         .unwrap();
     assert_eq!(matches.len(), 0);
-    drop_alert_from_collections(candid, "ZTF").await.unwrap();
 
-    // // Generate an LSST alert with the same ra and dec, but shifted north a bit within the search radius:
-    let (lsst_candid, _, _, _, lsst_bytes_content) = LsstAlertRandomizer::default()
+    // 2. nearby LSST alert, ZTF alert should have an LSST alias
+    let (_, lsst_object_id, _, _, lsst_bytes_content) = LsstAlertRandomizer::default()
         .ra(ra)
-        .dec(dec + 0.9 * r * 180.0 / std::f64::consts::PI)
+        .dec(dec + 0.9 * LSST_XMATCH_RADIUS.to_degrees())
         .get()
         .await;
     lsst_alert_worker
         .process_alert(&lsst_bytes_content)
         .await
         .unwrap();
 
-    // Now check again, the lsst_object_id should be present:
+    let (_, _, _, _, bytes_content) = ZtfAlertRandomizer::default()
+        .objectid(&object_id)
+        .ra(ra)
+        .dec(dec)
+        .get()
+        .await;
```

Comment:
Isn't this the same `bytes_content` as for the original random alert? I'm trying to figure out why `ZtfAlertRandomizer` needs to be run again here.

Reply:
No since I call default it randomized everything and then I keep the objectid/ra/dec identical, so basically only the candid is randomized again. The idea here is to generate a new candidates for the same object (and same position logically). We can't process the same alert (i.e. same candid) more than once so here we mimick generating a new alert for the same object.

Reply:
If we wanted the test to be cleaner, could just initialize the `ZtfAlerRandomizer` once at the top (and mutable), and just call `rand_candid` + `get` when we need a new alert. Let me do that now actually.

Reply:
done, required a bit of cloning but for testing I think that's fine.